pgcli/pgcompleter.py

```diff
@@ -384,38 +385,50 @@ def get_column_matches(self, suggestion, word_before_cursor):
 
         return self.find_matches(word_before_cursor, flat_cols, meta='column')
 
+    def generate_alias(self, tbl, tbls):
```

Comment:
I think this would be better off as a standalone function since it doesn't use any pgcompleter member variables. A docstring would help a lot.